man/systemd.socket.xml

```diff
@@ -429,6 +429,16 @@
         false, in read-only mode. Defaults to false.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ClearSocket=</varname></term>
+        <listitem><para>Takes a boolean argument. If true, the socket's buffers
+        are cleared before listening on the socket. The result will be that any pending
+        data will be terminated and any incoming connections will be terminated. If
```

Comment:
data isn't typically "terminated", but well, "flushed" or so.

And the incoming connections aren#t terminated either, since they were never accepted. Say "refused" or "rejected" or so. And maybe say *pending* incoming connections